synfig-studio/src/gui/app.cpp

```diff
@@ -4014,6 +4018,7 @@ App::new_instance()
 	canvas->rend_desc().set_w(preferred_x_size);
 	canvas->rend_desc().set_h(preferred_y_size);
 	canvas->rend_desc().set_antialias(1);
+	canvas->rend_desc().set_pixel_ratio(preferred_x_size, preferred_y_size);
```

Comment:
When you open a saved file, this will be set too?

Reply:
I think yes but we could force `toggled` signal to update pixel ratio. Need to test first.

If we don't set it for new documents we run into this issue https://github.com/synfig/synfig/pull/1895#issuecomment-768059119

Reply:
That was exactly what I was recalling, but I was unsure lol

Reply:
I still don't think this is the best way.

What do you think about only use `set_pixel_ratio()` when link is active (and set with the dimensions set at the moment link is clicked (to active)? In the same way, it would be reset to (1,1) when link is broken by user.

Reply:
I think ice0's new suggestion in this comment https://github.com/synfig/synfig/pull/1895#issuecomment-841949866 will let our dialog be untoggled by default. So we don't have to worry about forcing `set_pixel_ratio()` when link is inactive.